Sources/HTTP/Responder/HTTPScheme.swift

```diff
@@ -2,16 +2,17 @@
 public struct HTTPScheme {
     /// Plaintext data over TCP. Uses port `80` by default.
     public static var http: HTTPScheme {
-        return .init(80) { .done(on: $0.eventLoop) }
+        return .init(80) { channel, _ in .done(on: channel.eventLoop) }
     }
 
     /// Enables TLS (SSL). Uses port `443` by default.
     public static var https: HTTPScheme {
-        return .init(443) { channel in
+        return .init(443) { channel, hostname in
             return Future.flatMap(on: channel.eventLoop) {
                 let tlsConfiguration = TLSConfiguration.forClient(certificateVerification: .none)
```

Comment:
Yeah I think it would make sense for this to be `.fullVerification` since that's what NIO defaults too. I would just do `forClient()` so we get their defaults. However, I think this could be considered a breaking change so we should open a separate issue for this and discuss including it in the next major release.